CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Fixed
 - Command Generate - Avoid foreignKey to conflict with relationship.
 - Command Generate - Fix creation of project containing whitespaces.
+- Technical - Fix MySQL boolean values (Tinyint)
```

Comment:
```suggestion
- Records Update - Fix `Tinyint` column update when declared as a `boolean` field in the ORM model.
```
Seems better.
This is not a "Technical" change as it may impact the user experience.

Reply:
Thanks. I also changed the case of data types since it seems to be uppercase in MySQL doc.